Source/WebCore/Modules/websockets/WebSocketFrame.cpp

```diff
@@ -169,15 +168,14 @@ void WebSocketFrame::makeFrameData(Vector<uint8_t>& frameData)
     appendFramePayload(*this, frameData);
 }
 
-WebSocketFrame::WebSocketFrame(OpCode opCode, bool final, bool compress, bool masked, const uint8_t* payload, size_t payloadLength)
+WebSocketFrame::WebSocketFrame(OpCode opCode, bool final, bool compress, bool masked, std::span<const uint8_t> payload)
     : opCode(opCode)
     , final(final)
     , compress(compress)
     , reserved2(false)
     , reserved3(false)
     , masked(masked)
     , payload(payload)
```

Comment:
I keep thinking of `std::span<>` like an object, and wanting you to `WTFMove()` it.